code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -579,6 +579,14 @@ Please contact me on #coderbus IRC. ~Carnie x
 	var/standing	= image("icon"=female_clothing_icons["[t_color]_s"], "layer"=-layer)
 	return(standing)
 
+/mob/living/carbon/human/proc/get_overlays_copy(var/inhands=0)

```

Comment:
I think a *few* of them are also repeated, with incorrect numbers, in species.dm too.